### PR TITLE
2023 multilayer ec glioma

### PR DESCRIPTION
**Pull request by eduardacenteno**

### CODE REVIEW

File: multilayer_EC_glioma_2023/Readme.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## The longitudinal relation between executive functioning and multilayer network topology in glioma patients
+### Marike R van Lingen, Lucas C Breedt, Jeroen J G Geurts, Arjan Hillebrand, Martin Klein, Mathilde C M Kouwenhoven, Shanna D Kulik, Jaap C Reijneveld, Cornelis J Stam, Philip C De Witt Hamer, Mona L M Zimmermann, Fernando A N Santos, Linda Douw
+
+Scripts used in doi: 10.1007/s11682-023-00770-w.
```
Comment (eduardacenteno) [inline multilayer_EC_glioma_2023/Readme.md]: maybe make this a hyperlink?

File: multilayer_EC_glioma_2023/Supra_adjacencymatrix_github.m [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+%%% Reviewed by LC Breedt 2023/05/09 
+
+%%%%%%%%%%%%%%%%%%%%
+% Requirements     %
+%%%%%%%%%%%%%%%%%%%%
+
+%%% Other m-files required:
+% other m-files that are necessary to run the script, i.e. all functions etc.
+ 
+% /data/KNW/m.vanlingen/m2b_manuscript/fft_filt_BNA.m
```
Comment (eduardacenteno) [inline multilayer_EC_glioma_2023/Supra_adjacencymatrix_github.m]: maybe these don't need to be explicit paths in the server?

File: multilayer_EC_glioma_2023/adjusted_multilayer_main_code.py [hunk 3]
```diff
@@ -0,0 +1,854 @@
...
+
+#############################################
+# LOADING THE MATRICES                      #
+############################################# 
+
+# This is where the real data for computing all multilayer functions is loaded
+# Every function defined from now on uses these data as input
+
+# ATTENTION: THIS IS THE OBJECT THAT WILL BE USED FOR THE REMAINDER OF THE CODE!
+supra_mst = scipy.io.loadmat('/mnt/resource/m.vanlingen/m2b/matlab_output_final_okt2021/supra_mst_full.mat')
```
Comment (eduardacenteno) [inline multilayer_EC_glioma_2023/adjusted_multilayer_main_code.py]: explicit path can be removed here as well

File: multilayer_EC_glioma_2023/Readme.md (hunk 1 above)
Comment (eduardacenteno) [reply multilayer_EC_glioma_2023/Readme.md]: @mrvanlingen added some little changes to be made!

Comment (mrvanlingen) [reply multilayer_EC_glioma_2023/Readme.md]: Check :) Good suggestion!

File: multilayer_EC_glioma_2023/Supra_adjacencymatrix_github.m (hunk 2 above)
Comment (mrvanlingen) [reply multilayer_EC_glioma_2023/Supra_adjacencymatrix_github.m]: Totally right, didn't forgot to change those.

**Review by multinetlab — APPROVED** [review]
thanks for the changes, @mrvanlingen!

**Pull request merged**